app/src/main/java/org/fossasia/phimpme/leafpic/util/AlertDialogsHelper.java

```diff
@@ -61,6 +64,35 @@ public static AlertDialog getInsertTextDialog(final ThemedActivity activity, Ale
         dialogBuilder.setView(dialogLayout);
         return dialogBuilder.create();
     }
+    public static AlertDialog getInsertTextDialogLink(final ThemedActivity activity, AlertDialog.Builder dialogBuilder , EditText editText, String link) {
```

Comment:
You can modify the existing function as per your requirement. no need to create a new function again